Neos.TimeableNodeVisibility/Classes/Service/TimeableNodeVisibilityService.php

```diff
@@ -101,7 +98,7 @@ private function getNodesWithExceededDates(ContentRepository $contentRepository,
             $sitesNodeTypeName = NodeTypeName::fromString('Neos.Neos:Sites');
             $rootNode = $subgraph->findRootNodeByType($sitesNodeTypeName);
             if ($rootNode === null) {
-                throw RootNodeAggregateDoesNotExist::butWasExpectedTo($sitesNodeTypeName);
+                throw new \RuntimeException(sprintf('No sites root node found in content repository "%s"', $contentRepository->id->value), 1719047148);
```

Comment:
Just wondering: Why don't we keep the `RootNodeAggregateDoesNotExist` exception and throw it here? (not sure if it's needed but I'm curious about your reasons to replace it)

Reply:
we are in a different package here (the timeable stuff) and its imo not needed here to have a special exception class. Its a not ever gonna happen case and just a save guard.